cas-server-support-osf/src/main/java/io/cos/cas/adaptors/mongodb/OpenScienceFrameworkAuthenticationHandler.java

```diff
@@ -88,7 +89,7 @@
         @Field("is_registered")
         private Boolean isRegistered;
         @Field("merged_by")
-        private Boolean mergedBy;
+        private String mergedBy;
```

Comment:
Looks like that we mapped the wrong type before. This is probably due to OSF change that we were not informed of. Fortunately, it did not cause any issue because the logic is not reachable from normal authentication. I discovered this using a specially forged user.